app/lib/container/webcontainer-impl.ts

```diff
@@ -0,0 +1,74 @@
+import { WebContainer as OriginalWebContainer } from '@webcontainer/api';
```

Comment:
`WebContainer` doesn't seem to be used in this file, so we don't need aliasing as `OriginalWebContainer`.